### PR TITLE
add real sound player

### PR DESCRIPTION
**Pull request by SebastienPeillet**
@pblottiere This PR adds a real audio player in the qgis action.

### CODE REVIEW

**Comment by pblottiere** [comment]
Did you try on Windows too?

**Comment by SebastienPeillet** [comment]
@pblottiere not yet, I'll do it after lunch ;)

**Comment by SebastienPeillet** [comment]
@pblottiere It works like a charm  on windows too :ok_hand:

**Comment by pblottiere** [comment]
Awesome :tada:

File: src/core/layers/layer.py [hunk 1]
```diff
@@ -59,13 +60,205 @@ def _hideWidgetFid(self, layer: QgsVectorLayer) -> None:
...
+            timeformat(value) + " / " + timeformat(self.slider.maximum())
+        )
+
+    def end(self):
+        self.thread.time.disconnect(self.updateSlider)
+        self.thread.quit()
+        self.playButton.setChecked(False)
+        self.playing = False
+        self.slider.setSliderPosition(0)
+
```
Comment (pblottiere) [inline src/core/layers/layer.py:257]: Considering that the pyscript is pretty long now, could we move it in a dedicated file?

**Comment by pblottiere** [comment]
I just tested this PR on windows and it works well, thanks :+1:. Can you just take a look to this comment https://github.com/hytechimaging/sammo-boat/pull/130#pullrequestreview-844560604 please?

File: src/core/layers/layer.py (hunk 1 above)
Comment (SebastienPeillet) [reply src/core/layers/layer.py]: it's done ! ;)

**Comment by pblottiere** [comment]
Thanks!

**Pull request merged**